test/statistics_test.rb

```diff
@@ -9,5 +9,22 @@ def test_default_values
 
       assert Statistics.backtrace_cleaner.is_a?(Rails::BacktraceCleaner)
     end
+
+    def test_setting_show_top_sql_quesries
+      error = assert_raises(ArgumentError) do
+        Statistics.show_top_sql_queries = 'cat'
+      end
+      assert_match(/show_top_sql_queries must be one of:/, error.message)
+    end
+
+    def test_backtrace_cleaner
+      cleaner = Statistics.backtrace_cleaner
+      assert(cleaner.class, Rails::BacktraceCleaner)
+    end
+
+    def test_data
```

Comment:
You don't introduce the code you're testing here until the next commit. This test should be with it. Again, would've preferred you fixup/squash these kinds of issues before opening a PR.